CSS/explore.css

```diff
@@ -453,12 +456,27 @@ span.labelOncheck {
   #search2 {
     color: #cfd2e0;
   }
-  #oncolorClick {
```

Comment:
@beingharshil18 
This is also wrong practice, **#oncolorClick ** Use proper naming conventions